db/migrate/20240709020208_add_live_feedback_columns.rb

```diff
@@ -0,0 +1,7 @@
+class AddLiveFeedbackColumns < ActiveRecord::Migration[6.0]
+  def change
+    add_column :course_assessments, :live_feedback_enabled, :bool, null: false, default: false
+    add_column :course_assessment_question_programming, :live_feedback_enabled, :bool, null: false, default: false
+    add_column :course_assessment_question_programming, :live_feedback_custom_prompt, :string, null: false, default: ''
```

Comment:
Why not leave this as null? Otherwise we will not be able to implement a "fall-back" to default, vs user explicitly wants an empty prompt.

Reply:
I don't like using null as a value because it's not self-documenting in what it does, and it can cause things to behave unexpectedly down the line.